packages/ui-kit/src/components/code-block.js

```diff
@@ -202,14 +238,32 @@ const languageAliases = {
   js: 'javascript',
 };
 const CodeBlock = props => {
-  const languageCode = props.language || 'text';
+  const [content, setContent] = React.useState(
```

Comment:
Sorry but the state does not belong to the individual code block components, instead the parent component (the multi code block) should manage the state and render one or the other.